presidio-analyzer/app.py

```diff
@@ -82,7 +82,22 @@ def recognizers() -> Tuple[str, int]:
                     "during execution of "
                     "AnalyzerEngine.get_recognizers(). {}".format(e)
                 )
-                return json.dumps({"Error": e}), 500
+                return json.dumps({"Error": e.args}), 500
+
+        @self.app.route("/supportedentities", methods=["GET"])
+        def supported_entities() -> Tuple[str, int]:
```

Comment:
Is this new API documented> what should be the body etc. If not - let's open a user story/task and a TODO